### PR TITLE
single message background img style updates

### LINKED ISSUE #1173: update single msg background img styles
currently the image style used is large 1700 x 450, this should be updated to be 1700 x 500. 

See https://github.com/comicrelief/pattern-lab/issues/250

### test notes:
- go to https://pr-1174-3hxt5wy-tx3mbsqmxtu74.eu.platform.sh/fundraise
- check the image styles used for the background image single message row:
   - for the large breakpoint (1024px) the image should be cropped and scaled to 1700x500px
   - the medium and small breakpoints remain the same as they were (768x450px)

### PR DESCRIPTION
**Pull request by Heleen-cr**
Fixes #1173 
related to: https://github.com/comicrelief/pattern-lab/issues/250
## Changes proposed in this PR

- add large_1700x500 and large_1700x500_x2 image styles

### CODE REVIEW

File: modules/cr_rich_text_paragraph/cr_rich_text_paragraph.info.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ config_devel:
   - field.field.paragraph.cr_rich_text_paragraph.field_bg_single_msg_layout
   - field.field.paragraph.cr_rich_text_paragraph.field_background
   - field.field.paragraph.cr_rich_text_paragraph.field_body
-  - image.style.large_1700x500
-  - image.style.large_1700x500_2x
+  - image.style.large_1700x450
```
Comment (Saphyel) [inline modules/cr_rich_text_paragraph/cr_rich_text_paragraph.info.yml]: You shouldn't delete the old ones, that should be part of the remove old config ticket

**Comment by Saphyel** [comment]
Can you create a ticket for delete the old config from CR?

**Comment by Heleen-cr** [comment]
@Saphyel as you've seen already I've created a ticket in comicrelief to remove the old config and update the campaign version:
https://github.com/comicrelief/comicrelief/issues/347

**Comment by Heleen-cr** [comment]
passed QA

**Pull request merged**